src/main/java/me/damonkelley/tictactoe/BoardFormatter.java

```diff
@@ -0,0 +1,109 @@
+package me.damonkelley.tictactoe;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static java.util.Arrays.asList;
+
+public class BoardFormatter {
+    private Board board;
+
+    public BoardFormatter(Board board) {
+        this.board = board;
+    }
+
+    public List<List<Marker>> collect() {
+        List<List<Marker>> collection = new ArrayList<>();
```

Comment:
I think using the name collection here may be a bit confusing because collection is the interface which list is implementing.